uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1000.php

```diff
@@ -38,7 +38,7 @@ private function up1000_downloadUKEnglish() {
       mkdir($downloadPath, 0755, true);
     }
 
-    file_put_contents($downloadPath, fopen($localizationURL, 'r'));
+    file_put_contents($downloadPath . 'civicrm.mo', fopen($localizationURL, 'r'));
```

Comment:
This one is part of the upgrades done for this ticket, so I don't think it's a problem, since it hasn't been deployed to any live sites yet.